charts/prometheus-node-exporter/charts/prometheus-process-exporter/values.yaml

```diff
@@ -0,0 +1,133 @@
+# Default values for prometheus-process-exporter.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+image:
+  repository: ncabatoff/process-exporter
+  tag: 0.5.0
+  pullPolicy: IfNotPresent
+
+## Specify entries of `process_names:` in the process-exporter config
+## See https://github.com/ncabatoff/process-exporter/tree/master#using-a-config-file
+groups:
+- name: "{{.Comm}}"
+  cmdline:
+    - '.+'
+
+## Creates a Prometheus Operator ServiceMonitor
+serviceMonitor:
+  enabled: true
+  interval: "30s"
+  labels:
+    app.kubernetes.io/vendor: kubesphere
+
+service:
+  type: ClusterIP
+  port: 9201
+  targetPort: 9201
+  innerPort: 9202
+  nodePort:
+  annotations:
+    prometheus.io/scrape: "true"
+
+# Labels to add to the pod
+podLabels: {}
+
+# Annotations to add to the pod	
+podAnnotations: {}
+
+# Set to Kubernetes default updateStrategy by default.
+# See https://kubernetes.io/docs/tasks/manage-daemon/update-daemon-set/#daemonset-update-strategy for
+# supported configurations.
+updateStrategy:
+  type: RollingUpdate
+  rollingUpdate:
+    maxUnavailable: 1
+
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
```

Comment:
Process exporter's limit should be set to a larger value as https://github.com/kubesphere/kse-installer/blob/kse-3.x/roles/ks-monitor/templates/custom-values-process-exporter.yaml.j2#L7

Reply:
done. Do the resources of node-exporter need to be configured? What values ​​do we recommend?